carp.studies.core/src/commonMain/kotlin/dk/cachet/carp/studies/application/ParticipantServiceHost.kt

```diff
@@ -94,9 +119,9 @@ class ParticipantServiceHost(
         require( group.isNotEmpty() ) { "No participants to deploy specified." }
 
         // Verify whether the study is ready for deployment.
-        val study: Study = getStudyOrThrow( studyId )
-        check( study.canDeployToParticipants ) { "Study is not yet ready to be deployed to participants." }
-        val protocolSnapshot = study.protocolSnapshot!!
+        val recruitment = getRecruitmentOrThrow( studyId )
+        check( recruitment.canAddParticipations ) { "Study is not yet ready to be deployed to participants." }
```

Comment:
For consistency, this should maybe be renamed to `recruitment.isReadyForDeployment`? Essentially, it is after calling `recruitment.readyForDeployment` this returns true.